RFXtrx/__init__.py

```diff
@@ -703,6 +705,17 @@ def __init__(self, device, event_callback=None, debug=False,
     def _connect(self):
         """Connect """
         self.transport.reset()
+        self._status = self.send_get_status()
+
+        if self._modes is not None:
+            self.set_recmodes(self._modes)
+            self._status = self.send_get_status()
```

Comment:
Is it necessary to call send_get_status() both before and after calling set_recmodes() ?

Reply:
Yes, because the "set modes" command also includes the rfxtrx working frequency and output power. So we need to get their actual values before sending "set modes". For the moment I don't think it's necessary to be able to change them from the lib, but we may add the possibility later.

The second call is there to read the values that are actually set in the rfxtrx device (you could ask an invalid output power value or receiving mode which would not be set in reality).